docs/expand-pvc.md

```diff
@@ -66,16 +66,16 @@ metadata:
 - mounted Filesystem size in pod using this PVC
 
 ```bash
```

Comment:
Still bash? Maybe add that to the document too?

Reply:
Bash is still in use, I see here:
https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md040---fenced-code-blocks-should-have-a-language-specified